src/local/h2_adjust/units.py

```diff
@@ -1,15 +1,115 @@
 """
 Additional project specific units
 """
-from scmdata.units import UNIT_REGISTRY  # type: ignore
+from typing import Any
+
+import scmdata
+from scmdata.units import UNIT_REGISTRY, UnitConverter
+
+from local.h2_adjust.exceptions import SanitizeError
 
 if not hasattr(UNIT_REGISTRY, "hydrogen"):
     UNIT_REGISTRY.define("H = [hydrogen] = H2")
     UNIT_REGISTRY.define("hydrogen = H")
     UNIT_REGISTRY.define("t{symbol} = t * {symbol}".format(symbol="H"))
 
-    UNIT_REGISTRY.define("4/16 * H = CH4")
-    UNIT_REGISTRY.define(f"{14 + 2 * 16}H = NOx")
-    UNIT_REGISTRY.define("3/17 H = NH3")
+__all__ = [
+    "UNIT_REGISTRY",
+    "sanitize_combustion_intensity_units",
+    "sanitize_production_intensity_units",
+]
+
+
+def _get_unit_scaling(source_unit: str | Any, target_unit: str) -> float:
+    if not isinstance(source_unit, str):
+        return 1.0
```

Comment:
This feels unsafe to me. Any reason we can't raise then do try except higher up?

Reply:
I think this can be removed now after removing the 0's (previous comment)